super_csv/serializers.py

```diff
@@ -38,5 +40,15 @@ def get_related_queryset(cls, queryset):
         return queryset.select_related('user')
 
     def get_data(self, operation):
-        data = json.load(operation.data)
-        return CSVOperationDataSerializer(data).data
+        if bool(operation.data):
+            data = json.load(operation.data)
+            return CSVOperationDataSerializer(data).data
+        else:
+            error_message = 'Failed to retreive file'
+            logger.info(_(error_message))
+            return {
+                "total_rows": 0,
+                "processed_rows": 0,
+                "saved_rows": 0,
+                "error_message": error_message
```

Comment:
Translating here might make sense instead:

```suggestion
                "error_message": _('Failed to retrieve CSV operation data.')
```